src/Ledger/Ratify.lagda

```diff
@@ -345,6 +391,26 @@ DReps, regular DReps and SPOs.
   with the default depending on the action.
 \end{itemize}
 
+Let us discuss the last item above---the way SPO votes are counted---as the ledger
+specification's handling of this has evolved in response to community feedback.
+Previously, if an SPO did not vote, then they would be counted as having voted
+\abstain by default.  Members of the SPO community found this behavior counterintuitive
+and requested that non-voters be assigned a \no vote by default, with the caveat that
+an SPO could change their default setting by delegating their stake to an
```

Comment:
'their stake' here and 'if the SPO has delegated' below is too imprecise. This is specifically the delegation of the reward account, not the delegation of the pool credential. In particular, I think there should be a warning here that states that the credential used to vote is not (in general) the same as the one used to set the default. The credential that sets the default is the one used to withdraw staking rewards. And as a second layer, that means that if that credential is a script, it may need to have explicit logic written to be able to set a default at all.

Reply:
I've updated the description.  If you think it could be improved, please suggest further changes.  Thanks!